<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionBrightness_Modify_Sample"/>
    <addaction name="actionComing_soon"/>
    <addaction name="actionImage_Translation_Sample"/>
    <addaction name="actionDigital_Zoom_Sample"/>
    <addaction name="actionAlpha_Blend_Sample"/>
    <addaction name="actionUnsharp_Mask_Sample"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionComing_soon">
   <property name="text">
    <string>Contrast Modify Sample</string>
   </property>
  </action>
  <action name="actionBrightness_Modify_Sample">
   <property name="text">
    <string>Brightness Modify Sample</string>
   </property>
  </action>
  <action name="actionImage_Translation_Sample">
   <property name="text">
    <string>Image Translation Sample</string>
   </property>
  </action>
  <action name="actionDigital_Zoom_Sample">
   <property name="text">
    <string>Digital Zoom Sample</string>
   </property>
  </action>
  <action name="actionAlpha_Blend_Sample">
   <property name="text">
    <string>Alpha Blend Sample</string>
   </property>
  </action>
  <action name="actionUnsharp_Mask_Sample">
   <property name="text">
    <string>Unsharp Mask Sample</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
